File: hw/sivinta/U3/HW23Strategy/Strategy/src/main/java/ec/edu/espe/strategy/view/SortApp.java
package ec.edu.espe.strategy.view;

import ec.edu.espe.strategy.model.SortingContext;

/**
 *
 * @author Jahir Sivinta DCCO
 */
public class SortApp {

    public static void main(String args[]) {
        int data[] = {3, 6, 4, 6, 7, 8, 5, 6, 7, 5, 3, 3};

        System.out.print("Arreglo original: ");
        for (int num : data) {
            System.out.print(num + " ");
        }
        System.out.println();

        SortingContext sc = new SortingContext();
        int sortedList[] = sc.sort(data);
    }
}

File: ws/bonilla/U3/WS34Strategy/Strategyjs/view/SortApp.js
const SortingContext = require('../model/SortingContext');

function main() {
    const data = [3, 6, 4, 6, 7, 8, 5, 6, 7, 5, 3, 3];

    process.stdout.write("Arreglo original: ");
    data.forEach(num => process.stdout.write(num + " "));
    console.log();

    const sc = new SortingContext();
    const sortedList = sc.sort(data);
}

main();
